### PR TITLE
User routes

### PR DESCRIPTION
**Pull request by nverno**
I added some reducers and the users slice under entities in state.  Can you guys verify this layout is OK?

Test from console: `store.dispatch(userActions.fetchUsers())`
Test update: `store.dispatch(userActions.updateUser({ id: "5fe90d636ee7c9ca9f6f2d23", name: 'frodo' }))`

### CODE REVIEW

**Comment by nverno** [comment]
@kaycbas @YizheWill @oriravid

File: frontend/src/actions/user_actions.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  errors,
+});
+
+export const receiveUsers = (users) => ({
+  type: RECEIVE_USERS,
+  users,
+});
+
+export const fetchUsers = () => (dispatch) => {
+  UserAPI.fetchUsers()
```
Comment (kaycbas) [inline frontend/src/actions/user_actions.js]: Do we want to return the fetchUsers() promise here? I've occasionally found it useful to have in components

Comment (nverno) [reply frontend/src/actions/user_actions.js]: Good call- just a `return` right?

Comment (kaycbas) [reply frontend/src/actions/user_actions.js]: yeah

**Pull request merged**